### PR TITLE
91049 add travel reimbursement v2 settings

### PR DESCRIPTION
**Pull request by randomsync**
## Summary
This PR adds settings for the travel pay api cutover to new endpoint with new api credentials. Accompanying changes in config repo: https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3074

- *This work is behind a feature toggle (flipper): YES*

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/91049

## Testing done

- [x] test locallly
- [ ] will be testing in staging with feature flag turned on once deployed

## What areas of the site does it impact?
check-in experience travel claims submission and status check functionality

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).

### CODE REVIEW

**Comment by randomsync** [comment]
working on adding changes to configs

**Review by kanchanasuriya — APPROVED** [review]
LGTM

File: config/settings.yml [hunk 1]
```diff
@@ -1784,4 +1795,4 @@ schema_contract:
   test_index: 'spec/fixtures/schema_contract/test_schema.json'
 
 form_mock_ae_design_patterns:
-  prefill: true
\ No newline at end of file
+  prefill: true
```
Comment (mdewey) [inline config/settings.yml:1798]: Was this meant to be added?

Comment (randomsync) [reply config/settings.yml:1798]: a newline at the end of file was automatically added by the IDE, guess for no other reason than a historical standard: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725#729725.

**Review by ericboehs — APPROVED** [review]

**Pull request merged**